src/test/run-pass/auxiliary/arc_wake.rs

```diff
@@ -8,13 +8,14 @@ use std::task::{
 };
 
 macro_rules! waker_vtable {
```

Comment:
If I am not mistaken, this macro is never used since it would no longer produce code that compiles. The field names would have to be removed now that it is a function call instead of a struct initializer.

Reply:
They have been removed.